raspyrfm_client/device/manufacturer/hama/_121938.py

```diff
@@ -0,0 +1,47 @@
+from raspyrfm_client.device import actions
+from raspyrfm_client.device.base import Device
+
+
+class HAMA(Device):
```

Comment:
Yes this seems like a good idea. This will hopefully make it easy to support other gateways down the road.